draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,220 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Cooperative Scenario}{Close to enemies}{\images/balistic.png}
+
+\begin{multicols}{2}
+
+\textbf{Author:} Kelghor
+
+\textbf{Version:} 1.5
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1313651865648369725}{Archon Studio Discord}
+
+\textit{"Sir! They're here again!"}
+
+\textit{"Sound the alarm, we'll repel them as always!" you command, and go to defend the city. Just as you have done many times before. Such is the hard life of the surrounding clans, with constant wars and skirmishes with nearby neighbors. Fortunately, your alliance is strong, so you know your back is covered.}
+
+\textit{Therefore, the time has come to end the long-standing wars and crush your enemies like a bug once and for all!}
+
+\subsection*{\MakeUppercase{Scenario Length}}
+
+This Scenario is played over 7 Rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+
+\textbf{Player Count:} 1-6
+
+\textbf{Starting Resources:} 14 \svg{gold}, 5 \svg{building_materials}, 2 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item  1 × A Few \svgunit{bronze} Units of your choice
+\end{itemize}
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling.
+
+\textbf{Additional Bonus:}: Only base bonus by difficulty (see the main rules).
+
+\textbf{Map Tile Pool:} Take $P$ × random Far (II–III) map Tiles as shared pool. None of them must contain a settlement. ($P$ stands for the number of players)
+
+\subsection*{\MakeUppercase{Map Setup}}
+
+Take the following Map Tiles and arrange them as shown in the Scenario map layout.
+Setup Far (II--III) Map Tiles with settlements of same Faction as players Towns as show a color Cube, or see example.
+
+\begin{itemize}
+  \item $P$ × Starting (I) Map Tile.
+  \item $P$ × Far (II--III) Map Tile with Settlement of same Faction as players Towns.
+  \item 2$P$ × Far (II--III) Map Tile - must not contain Tiles with Settlements.
+\end{itemize}
```

Comment:
@Dima793 Little help here?

Reply:
What problem is here? :-)

Reply:
@Dima793 changed the formatting around this recently in other scenarios. He might have an idea how to keep it consistent.

Reply:
Current version doesn't compile for me so can't check, but I have two options to suggest:

Reply:
Option 1
```suggestion
\begin{itemize}
  \item P × Starting (I) Map Tile.
  \item P × Far (II--III) Map Tile with the Settlement Field of the same Faction as player's Town.
  \item 2P × Far (II--III) Map Tile without the Settlement Field.
\end{itemize}
```

Reply:
Option 2
```suggestion
\begin{itemize}
  \item P × Starting (I) Map Tile
  \item 3P × Far (II--III) Map Tile, exactly P of which must contain a Settlement of the same Faction as player's Town
\end{itemize}
```

Reply:
In either option you can remove the "of the same Faction as player's Town" condition from the tile list if you think it looks too bulky, and add it as a note below, similar to the note below map tile list in the Titans' Stronghold scenario. But it affects tile choice rather than tile placement, so I think having it said within the tile list (as it is now) would be more consistent with how such conditions are arranged in other scenarios.